tests/unit/test_errors.py

```diff
@@ -0,0 +1,97 @@
+"""Module testing error responses."""
+
+import pytest
+
+from eq_cir_management_ui import create_app
+from eq_cir_management_ui.config import config
+
+
+@pytest.fixture(name="test_client")
+def create_client():
+    """Creates and configures a test client for the application.
+
+    This function initializes the application in testing mode and provides
+    a test client that can be used to simulate HTTP requests during unit tests.
+
+    Yields:
+        FlaskClient: A test client instance for the application.
+    """
+    app = create_app(config.DefaultConfig)
+    app.config["TESTING"] = True
+
+    with app.test_client() as client:
+        yield client
+
+
+def test_error_404_route(test_client):
```

Comment:
We should use pytest features and "parametrize" into one test, remove all others, since they all test the same thing. http://docs.pytest.org/en/stable/example/parametrize.html